packages/modules/customer/src/models/customer-group.ts

```diff
@@ -1,21 +1,33 @@
 import { DAL } from "@medusajs/types"
-import { DALUtils, Searchable, generateEntityId } from "@medusajs/utils"
+import {
+  DALUtils,
+  Searchable,
+  createPsqlIndexStatementHelper,
+  generateEntityId,
+} from "@medusajs/utils"
 import {
   BeforeCreate,
+  Collection,
   Entity,
+  Filter,
+  ManyToMany,
   OnInit,
   OptionalProps,
   PrimaryKey,
   Property,
-  ManyToMany,
-  Collection,
-  Filter,
 } from "@mikro-orm/core"
 import Customer from "./customer"
 import CustomerGroupCustomer from "./customer-group-customer"
 
 type OptionalGroupProps = DAL.SoftDeletableEntityDateColumns // TODO: To be revisited when more clear
 
+const CustomerGroupUniqueName = createPsqlIndexStatementHelper({
```

Comment:
Should we maybe normalize the group name to be case-insensitive?

Reply:
To me names are a bit different from e.g. handles, and therefore I think we can have looser restrictions for them. 

E.g. I would be fine with having "VIP Customers" and "vip customers".